docs/content/en/docs/intro/_index.md

```diff
@@ -1,173 +1,124 @@
-on the production site.
 
-Keptn includes multiple features
-that can be implemented independently or together.
-Different features are at different levels of stability.
-See the [Keptn README file](https://github.com/keptn/lifecycle-toolkit/blob/main/README.md)
-for a list of the features that have been implemented to date
-and their level of stability.
-
-In a December 2022 Keptn Community meeting,
-we discussed the differences and similarities
-between Keptn v1 and Keptn
-to help you decide which best fits your needs.
-View the recording:
-[Compare Keptn v1 and the Keptn Lifecycle Toolkit](https://www.youtube.com/watch?v=-cKyUKFjtwE&t=170s)
-
-## Overviews of Keptn
-
-A number of presentations are available to help you understand Keptn.
-Note that many of these refer to the "Keptn Lifecycle Controller"
-or "Keptn Lifecycle Toolkit", which was the development code name for Keptn.
-
-* [Orchestrating and Observing GitOps Deployments with Keptn](https://www.youtube.com/watch?v=-cKyUKFjtwE&t=11s)
-  discusses the evolution of Keptn and the concepts that drive it,
-  then gives a simple demonstration of a Keptn implementation.
-
-* [Introducing Keptn Lifecycle Toolkit](https://youtu.be/449HAFYkUlY)
-  gives an overview of what Keptn does and how to implement it.
-
-* [Keptn Lifecycle Toolkit Demo Tutorial on k3s, with ArgoCD for GitOps, OTel, Prometheus and Grafana](https://www.youtube.com/watch?v=6J_RzpmXoCc)
-  is a short video demonstration of how Keptn works.
-  You can download the exercise and run it for yourself;
-  notes below the video give a link to the github repo.
-  The README file in that repo gives instructions for installing the software
-  either automatically or manually.
-
-* [What is the Keptn Lifecycle Toolkit?](https://isitobservable.io/observability/kubernetes/what-is-the-keptn-lifecycle-toolkit)
-  blog discusses Keptn as part of the "Is It Observable?" series.
-  This links to:
-
-* [What is Keptn Lifecycle Toolkit?](https://www.youtube.com/watch?v=Uvg4uG8AbFg)
-  is a video that steps through the process of integrating Keptn
-  with your existing cloud native cluster.
-
-* [Keptn Lifecycle Toolkit: Installation and KeptnTask Creation in Minutes](https://www.youtube.com/watch?v=Hh01bBwZ_qM)
-  demonstrates how to install Keptn and create your first KeptnTask in less than ten minutes.
-  
-* You can explore the [GitHub repository](https://github.com/isItObservable/keptn-lifecycle-Toolkit)
-  for more information.
+## Release lifecycle management
+
+The Release lifecycle management tools run in conjunction
+with the standard Kubernetes deployment tools
+to make deployments more robust.
+
+These tools run checks and tasks before or after deployment initiation.
+
+* Pre-deployment tasks such as checking for dependent services,
+  image scanning, and setting the cluster to be ready for the deployment
+
+* Pre-deployment evaluations such as checking the layout of the cluster
```

Comment:
maybe change this to "checking if there are enough resources in your cluster"
checking the layout of a cluster is not really a thing that people do IMO

Reply:
I wondered about that.  I stole that phrase from somewhere but yours is much better -- done.